datamap/bloom/src/main/java/org/apache/carbondata/datamap/bloom/BloomCoarseGrainDataMapFactory.java

```diff
@@ -66,22 +66,32 @@
    */
   private static final String BLOOM_SIZE = "bloom_size";
   /**
-   * default size for bloom filter: suppose one blocklet contains 20 pages
-   * and all the indexed value is distinct.
+   * default size for bloom filter, cardinality of the column.
    */
-  private static final int DEFAULT_BLOOM_FILTER_SIZE = 32000 * 20;
+  private static final int DEFAULT_BLOOM_FILTER_SIZE = Short.MAX_VALUE;
```

Comment:
Can you make a page size constant and use it, so that we can easily change it later when we make page size configurable

Reply:
ok